vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/model/AppHealthState.kt

```diff
@@ -29,6 +29,7 @@ data class AppHealthState(
     val alerts: List<String>,
     val healthDataJsonString: String,
     val restartedAtEpochSeconds: Long?,
+    val badHealthStartEpochSeconds: Long? = null,
```

Comment:
You're checking `this.badHealthStartEpochSeconds ?: return -1`, why not setting this as -1?

Reply:
Yeah, also could do that. The kotlin code for that check is simpler when its `null`.
```kotlin
val startSeconds = this.badHealthStartEpochSeconds ?: return -1
```

vs

```kotlin
val startSeconds = if (this.badHealthStartEpochSeconds == -1) {...} else {...}
```